src/extension/importer/core/src/main/java/org/geoserver/importer/Directory.java

```diff
@@ -39,7 +39,7 @@
 public class Directory extends FileData {
 
     private static final Logger LOGGER = Logging.getLogger(Directory.class);
-    
+
```

Comment:
In this file I only see reformats, but might be wrong. Formats changes are not encouraged, and if you have to do them for any reason, place them in a different commit.